getimage.py

```diff
@@ -0,0 +1,48 @@
+import httplib, urllib, base64, random, string, json, pprint, cStringIO
+from PIL import Image
+import io 
+
+def get_random_image(): 
+    search_term = "".join(random.choice(string.ascii_lowercase) for j in range(random.randint(1,6)))
+
+    headers = {
+        'Ocp-Apim-Subscription-Key': '9eca18ad3a9f4d16990d07cf0b86221b',
+    }
+
+    params = urllib.urlencode({
+        'q': search_term
+    })
+    try:
+        conn = httplib.HTTPSConnection('api.cognitive.microsoft.com')
+        conn.request("POST", "/bing/v5.0/images/search?%s" % params, "{body}", headers)
+        response = conn.getresponse()
+        data = response.read()
+        conn.close()
+        image_json = json.loads(data)
+        match_number = random.randint(0, len(image_json['value'])-1)
+        url = image_json['value'][match_number]['contentUrl']
+        img_file = io.BytesIO(urllib.urlopen(url).read())
+        img = Image.open(img_file)
+        img.show()
+    except Exception as e:
```

Comment:
**Exception** is the base exception class and will catch all exceptions which inherit from it. It's better to have than not when dealing with non determinism, however when you are consume **Exception** you abstract some details which might be viable for debugging.